test/test_checkers.py

```diff
@@ -145,12 +145,54 @@ class MyFakeChecker(Checker):
         VENDOR_PRODUCT = [("my", "checker")]
         IGNORE_PATTERNS = [r"mychecker-5.6"]
 
-    string = "Some other lines. \n Ignore this version pattern mychecker-5.6. \n Consider this version pattern mychecker-5.8. \n Some more lines."
-    lines = string.split("\n")
-    checker = MyFakeChecker()
-
-    result1 = checker.get_versions(lines[1], "")
-    assert result1.versions[0] == "UNKNOWN"
-
-    result2 = checker.get_versions(lines[2], "")
-    assert result2.versions[0] == "5.8"
+    def test_multiple_versions(self):
```

Comment:
We should modify this test case to pass all of the lines into the checker at once, instead of one at a time. The point of this test case is to ensure that one result object will contain both of the version numbers `5.8` and `6.0`.